File: PHP/salary/person.php
<?php
abstract class Person {
    protected int $identification;
    protected string $name;
    protected string $lastName;

    public function __construct(int $identification, string $name, string $lastName) {
        $this->identification = $identification;
        $this->name = $name;
        $this->lastName = $lastName;
    }

    public function getIdentification(): int {
        return $this->identification;
    }

    public function getName(): string {
        return $this->name;
    }

    public function getLastName(): string {
        return $this->lastName;
    }

    abstract public function calculateSalary(): float;
}
?>

File: Java/salary/person.java
public abstract class person {
    private final int identification;
    private final String name;
    private final String lastName;

    public person(int identification, String name, String lastName) {
        this.identification = identification;
        this.name = name;
        this.lastName = lastName;
    }

    public int getIdentification() {
        return this.identification;
    }

    public String getName() {
        return this.name;
    }

    public String getLastName() {
        return this.lastName;
    }

    public abstract double calculateSalary();
}